modules/critical-sev-security-policies.adoc

```diff
@@ -20,6 +20,7 @@ The following table lists the default security policies in {product-title} that
 |Life cycle stage |Name |Description | Status
 |Build or Deploy |Apache Struts: CVE-2017-5638|Alerts when deployments have images that contain the CVE-2017-5638 Apache Struts vulnerability. |Enabled
 |Build or Deploy |Log4Shell: log4j Remote Code Execution vulnerability |Alerts when deployments include images that contain the CVE-2021-44228 and CVE-2021-45046 Log4Shell vulnerabilities. Flaws exist in the Apache Log4j Java logging library in versions 2.0-beta9 - 2.15.0, excluding version 2.12.2. |Enabled
+|Build or Deploy |Rapid Reset: Denial of Service Vulnerability in HTTP/2 Protocol |Alerts on deployments with images containing components that are susceptible to a Denial of Service (DoS) vulnerability for HTTP/2 servers. This addresses a flaw in the handling of multiplexed streams in HTTP/2. A client can rapidly create a request and immediately reset them, which creates extra work for the server while avoiding hitting any server-side limits, resulting in a denial of service attack. |Enabled
```

Comment:
@kcarmichael08 We are in the process of disabling this policy out of the box. I'll DM you the slack thread.

Reply:
Yes we should add it. But the added text says "Enabled". We'd like to change that to disabled?

Reply:
So I should wait to add it until after 4.4.4? Because it's currently enabled, correct?

Reply:
(The ticket is to change it to disabled and change the severity category for 4.4.4+. If it makes sense to wait until that release, it's OK, but the release isn't until sometime after July 1.)

Reply:
Aah I see. Yes it is currently enabled. It will be disabled starting 4.5. I don't believe the 4.4 train will change.